src/herder/test/HerderTests.cpp

```diff
@@ -1451,7 +1451,7 @@ TEST_CASE("SCP Driver", "[herder][acceptance]")
 {
     SECTION("protocol 10")
     {
-        testSCPDriver(10, 10, 10, false, false);
+        testSCPDriver(10, 10, 10, false, true);
```

Comment:
this section should be removed (in the context of this commit)

Reply:
I've removed it.  That also made the `checkHighFee` parameter obsolete (all remaining calls to `testSCPDriver()` used `true`), so I removed that too.